demo/xml-output/xml-example.js

```diff
@@ -0,0 +1,17 @@
+// example xml output
+var out = outputXMLWithDoctype("demo/xml-output/xml-example.xml", "some_system.dtd");
+out.xml().beginResult("root-element");
```

Comment:
why not return a StructuredOutput here and make the "attr" function static / available in JS as xmlAttribute or even better as fluent helper method